src/VisualStudio/Core/Def/Packaging/PackageSearchService.cs

```diff
@@ -94,12 +91,49 @@ private static IPackageSearchRemoteControlService CreateRemoteControlService(VSS
 
             _cacheDirectoryInfo = new DirectoryInfo(Path.Combine(
                 localSettingsDirectory, "PackageCache", string.Format(Invariant($"Format{_dataFormatVersion}"))));
-            // _databaseFileInfo = new FileInfo(Path.Combine(_cacheDirectoryInfo.FullName, "NuGetCache.txt"));
 
             _cancellationTokenSource = cancellationTokenSource;
             _cancellationToken = _cancellationTokenSource.Token;
         }
 
+        internal void Start()
+        {
+            var options = _workspace.Options;
+            if (!options.GetOption(ServiceComponentOnOffOptions.PackageSearch))
+            {
+                return;
+            }
+
+            // Start the whole process once we're connected
+            _installerService.PackageSourcesChanged += OnPackageSourcesChanged;
+            OnPackageSourcesChanged(this, EventArgs.Empty);
+            _started = true;
```

Comment:
Nope.  Scratch that.  Still necessary.

This boolean is just so we know if we've actually done the event hookup.  That way we know if we should disconnect on 'Stop' down below.  Where it comes here shouldn't really matter.